client/blocks/plan-thank-you-card/style.scss

```diff
@@ -0,0 +1,164 @@
+.plan-thank-you-card .thank-you-card__main-icon {
+	margin-bottom: 8px;
+}
+
+.plan-thank-you-card.is-free-plan {
+	.thank-you-card{
```

Comment:
I'm wondering if we can create mixin that would take care of all color changes to simplify this file.